### PR TITLE
Add section on exploring proveneance

### PR DESCRIPTION
**Pull request by chrisjsewell**
Small addition to quickly merge 😬 
In the current notebook, it references an appendix section that doesn't exist. Also there was no direct link to the `exploring_provenance` appendix.
So I add that below, plus a small section on the `Graph` API.

Obviously I have tested these outputs in the "solution" notebook. This is what it looks like:

image
image

### CODE REVIEW

**Comment by chrisjsewell** [comment]
and this is what it looks like in the documentation: https://aiida-tutorials--261.org.readthedocs.build/en/261/pages/2020_Intro_Week/notebooks/querybuilder-tutorial.html#generating-a-provenance-graph

File: docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md
Comment (mbercx) [inline docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md]:
```suggestion
To do this, AiiDA actually uses some of the queries which you have learned about above.
```

File: docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md
Comment (mbercx) [inline docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md]:
```suggestion
For example, let's query for a calculation, then use methods of the `Graph` class to visualise the inputs and outputs of this calculation:
```

File: docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md
Comment (mbercx) [inline docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md]:
```suggestion
The `Graph` class also has methods for recursing up or down the provenance tree.
```

File: docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md
Comment (mbercx) [inline docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md]:
```suggestion
In this example, let's query for a pseudopotential, and visualise which processes it is used in:
```

File: docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md
Comment (mbercx) [inline docs/pages/2020_Intro_Week/notebooks/querybuilder-template.md]:
```suggestion
For further information on using `Graph` to generate provenance graphs, please see [this section](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/visualising_graphs/visualising_graphs.html) in the documentation.
```

**Review by mbercx — CHANGES_REQUESTED** [review]
Thanks @chrisjsewell. Seems fine overall to me, save for some typos. I'll leave it to @CasperWA to merge, as he is the session chair.

**Comment by chrisjsewell** [comment]
Thanks @mbercx, I clearly forget how to spell at 6 in the morning lol.
Well I'm going to merge this soon if @CasperWA is still sleeping 😜

**Review by yakutovicha — APPROVED** [review]
Looks good!

**Pull request merged**
